### PR TITLE
Endpoints JSON File

### PR DESCRIPTION
**Pull request by HanzoDaBoss**
Created GET endpoints for league tables, fixtures, teams, and team sheets

Created GET endpoints for venues

Created GET, POST, PATCH, and DELETE endpoints for users

### CODE REVIEW

File: endpoints.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                    {"team_name": "Old Bags", "points": 38, "wins": 10, "draws": 4, "losses": 6,"goals_for": 102, "goals_against": 77}
+                ]
+            }
+        ]
+      }
+    },
+    "GET /api/fixtures": {
+        "description": "serves an array of all fixtures",
+        "exampleResponse": {
+            "fixtures": [
```
Comment (Stephen0wen) [inline endpoints.json]: Needs division query

File: endpoints.json [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    },
+    "GET /api/fixtures/:fixture_id/team_sheet": {
+        "description": "serves an team sheet object corresponding to the given fixture id",
+        "exampleResponse": {
+            "team_sheet":
+                {
+                "fixture_id": 1,
+                "Team1_name": "Leicester Wolves",
+                "Team2_name": "Old Bags",
+                "Team1_players": [
```
Comment (Stephen0wen) [inline endpoints.json]: Needs Times

File: endpoints.json [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            }
+        }
+    },
+    "GET /api/fixtures/:fixture_id/team_sheet": {
+        "description": "serves an team sheet object corresponding to the given fixture id",
+        "exampleResponse": {
+            "team_sheet":
+                {
+                "fixture_id": 1,
+                "Team1_name": "Leicester Wolves",
```
Comment (Stephen0wen) [inline endpoints.json]: Team id?

File: endpoints.json [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+                  {"team_name": "Leicester Wolves", "points": 45, "wins": 12, "draws": 4, "losses": 4,"goals_for": 108, "goals_against": 67},
+                  {"team_name": "Old Bags", "points": 38, "wins": 10, "draws": 4, "losses": 6,"goals_for": 102, "goals_against": 77}
+              ]
+          }
+      ]
+    }
+  },
+  "GET /api/fixtures": {
+      "description": "serves an array of all fixtures",
+      "queries": ["status", "player", "team"],
```
Comment (Stephen0wen) [inline endpoints.json]: Division

File: endpoints.json [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+  "GET /api/fixtures/:fixture_id/team_sheet": {
+      "description": "serves an team sheet object corresponding to the given fixture id",
+      "exampleResponse": {
+          "team_sheet":
+              {
+              "fixture_id": 1,
+              "Team1_name": "Leicester Wolves",
+              "Team2_name": "Old Bags",
+              "Team1_players": [
+                  {"name": "Hanif", "person_id": 1, "availability": "yes"},
```
Comment (Stephen0wen) [inline endpoints.json]: Date / Time

File: endpoints.json [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+          }
+      }
+  },
+  "GET /api/fixtures/:fixture_id/team_sheet": {
+      "description": "serves an team sheet object corresponding to the given fixture id",
+      "exampleResponse": {
+          "team_sheet":
+              {
+              "fixture_id": 1,
+              "Team1_name": "Leicester Wolves",
```
Comment (Stephen0wen) [inline endpoints.json]: Team ids?

File: endpoints.json [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+      "exampleResponse": {
+          "fixture":
+              {
+              "fixture_id": 1,
+              "match_status": "created",
+              "Team1_name": "Leicester Wolves",
+              "Team2_name": "Old Bags",
+              "Team1_score": 20, 
+              "Team2_score": 10,
+              "venue_name": "Mior Sufian Centre",
```
Comment (Stephen0wen) [inline endpoints.json]: Venue id

File: endpoints.json [hunk 8]
```diff
@@ -0,0 +1,255 @@
...
+      "description": "serves an array of all users",
+      "queries": [],
+      "exampleResponse": {
+        "users": [
+            {
+              "person_id": 1,
+              "team_id": 2,
+              "person_name": "Alfie Fenables",
+              "team_name": "Leicester Wolves", 
+              "person_role": "Secretary",
```
Comment (Stephen0wen) [inline endpoints.json]: Array of "true" roles

File: endpoints.json [hunk 9]
```diff
@@ -0,0 +1,255 @@
...
+            }
+      }
+    },
+    "GET /api/users": {
+      "description": "serves an array of all users",
+      "queries": [],
+      "exampleResponse": {
+        "users": [
+            {
+              "person_id": 1,
```
Comment (Stephen0wen) [inline endpoints.json]: user_id not person

File: endpoints.json [hunk 10]
```diff
@@ -0,0 +1,255 @@
...
+      "queries": [],
+      "exampleResponse": {
+        "users": 
+            {
+              "person_id": 1,
+              "team_id": 2,
+              "person_name": "Alfie Fenables",
+              "team_name": "Leicester Wolves", 
+              "person_role": "Secretary",
+              "person_postcode": "BR23 2XU",
```
Comment (Stephen0wen) [inline endpoints.json]: Array of roles

File: endpoints.json [hunk 11]
```diff
@@ -0,0 +1,255 @@
...
+      "queries": [],
+      "exampleResponse": {}
+    },
+    "PATCH /api/users/:user_id": {
+      "description": "updates and serves a user object corresponding to the given user id",
+      "queries": [],
+      "exampleRequest": {
+          "body": {
+              "person_postcode": "BR24 8XY", 
+              "team_name": "Bedouins"
```
Comment (Stephen0wen) [inline endpoints.json]: Full Address

File: endpoints.json [hunk 12]
```diff
@@ -0,0 +1,255 @@
...
+      "description": "inserts and serves a user object corresponding to the given user id",
+      "queries": [],
+      "exampleRequest": {
+          "body": 
+          {
+            "person_name": "Alfie Fenables",
+            "team_name": "Leicester Wolves", 
+            "person_role": "Secretary",
+            "person_postcode": "BR23 2XU",
+            "person_dob": "15/02/2000"
```
Comment (Stephen0wen) [inline endpoints.json]: Full Address

**Review by Stephen0wen — CHANGES_REQUESTED** [review]
Cool beans

File: endpoints.json (hunk 1 above)
Comment (miors) [reply endpoints.json]: added

File: endpoints.json (hunk 2 above)
Comment (miors) [reply endpoints.json]: added

File: endpoints.json (hunk 3 above)
Comment (miors) [reply endpoints.json]: added

File: endpoints.json (hunk 4 above)
Comment (miors) [reply endpoints.json]: added

File: endpoints.json (hunk 5 above)
Comment (miors) [reply endpoints.json]: added

File: endpoints.json (hunk 6 above)
Comment (miors) [reply endpoints.json]: added

File: endpoints.json (hunk 7 above)
Comment (miors) [reply endpoints.json]: added

File: endpoints.json (hunk 8 above)
Comment (miors) [reply endpoints.json]: added

File: endpoints.json (hunk 9 above)
Comment (miors) [reply endpoints.json]: changed to user_id

File: endpoints.json (hunk 10 above)
Comment (miors) [reply endpoints.json]: changed

File: endpoints.json (hunk 11 above)
Comment (miors) [reply endpoints.json]: added

File: endpoints.json (hunk 12 above)
Comment (miors) [reply endpoints.json]: added

**Comment by miors** [comment]
Hi @Stephen0wen. We have updated the endpoints. Could you have a look and merge accordingly please. Thank you. cc: @HanzoDaBoss

**Review by Stephen0wen — APPROVED** [review]

**Pull request merged**